### PR TITLE
fix: page transition with query string on edit page does not work

### LINKED ISSUE #319: Page transition with query string on edit page does not work
## Description

In the detail page, when there are no tenants and the "Add Tenants" button is clicked, the user should be redirected to the tenant tab of the edit page. However, this functionality is currently not implemented, and users are instead taken to the default task tab.

This can be easily implemented by reusing the existing logic from the detail view.

## To Reproduce

1. Go to detail page of the share house that has no tenant
2. Click "Add Tenants" button

## Screenshots or videos

image
image

### PR DESCRIPTION
**Pull request by KaiKoide**
## Overview

Implement logic to change a redirection based on query parameters.

## Screenshots or videos

https://github.com/user-attachments/assets/e8c2c3e5-7776-4ba9-a346-4a7dcd50b261

## Assignee Checklist:

- [ ] The base branch is correct (no accidental merges)
- [ ] The branch name follows our branch naming rules
- [ ] The PR title follows our PR title rules
- [ ] My code follows our coding style
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [x] Code readability and simplicity
- [ ] Follows best practices and coding standards
- [x] Understandable and maintainable code

### CODE REVIEW

File: src/components/edit/EditTabsManager.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use client';
+import React from 'react';
+import { useSearchParams, usePathname, useRouter } from 'next/navigation';
+
+import { Tabs, TabsList, TabsTrigger } from '@/components/ui/tabs';
+
+export default function EditTabsManager({
```
Comment (yuki-92c) [inline src/components/edit/EditTabsManager.tsx]: Please use named exports instead of default to follow Coding conventions

**Review by yuki-92c — COMMENTED** [review]
Thank you Kai, Please check the comment!

File: src/components/edit/EditTabsManager.tsx (hunk 1 above)
Comment (KaiKoide) [reply src/components/edit/EditTabsManager.tsx]: I changed an export default to named export.

Comment (yuki-92c) [reply src/components/edit/EditTabsManager.tsx]: Thank you LGTM!

**Review by yuki-92c — APPROVED** [review]
Thank you Kai, Please check the comment!

**Pull request merged**